transforms/ember-object/__testfixtures__/options/decorators-disabled.input.js

```diff
@@ -4,74 +4,74 @@ import { inject as service } from "@ember/service";
 import { on } from "@ember/object/evented";
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo1 = EmberObject.extend({
   computed: computed({
     get() {},
     set() {}
   })
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo2 = EmberObject.extend({
   statefulObject: {},
   statefulArray: []
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo3 = EmberObject.extend({
   tagName: "div"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo4 = EmberObject.extend({
   classNames: []
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo5 = EmberObject.extend({
   classNameBindings: ["foo"],
   foo: "val"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo6 = EmberObject.extend({
   attributeBindings: ["foo"],
   foo: "val"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo7 = EmberObject.extend({
   actions: {
     bar() {}
   }
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo8 = EmberObject.extend({
   observer: observer()
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo9 = EmberObject.extend({
   event: on()
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo10 = EmberObject.extend({
   event: service()
```

Comment:
I don't get why some of these aren't transformed. I know that these changes are just dealing with
some looseness in the old parser (allowing duplicate `const`s), but do you know the answer?

Reply:
This could be `@service event` in the cases where decorators are not disabled (as they are in this test).